hideIt.sh

```diff
@@ -673,7 +664,8 @@ function main() {
         _WAIT_PID=$!
     elif [ $SIGNAL -eq 0 ]; then
         printf "Waiting for SIGUSR1...\n"
-        serve_signal &
```

Comment:
Because `serve_signal` was a subprocess, the parent process didn't have a handler for `SIGUSR1`. This caused it to exit when it received a `SIGUSR1` signal. This now gets handled correctly.